SQL/New_patches/2018-05-28_publication_schema.sql

```diff
@@ -0,0 +1,174 @@
+CREATE TABLE `publication_collaborator_rel` (
+  `PublicationID` int(10) unsigned NOT NULL,
+  `PublicationCollaboratorID` int(10) unsigned NOT NULL,
+  CONSTRAINT `PK_publication_collaborator_rel` PRIMARY KEY(PublicationID, PublicationCollaboratorID),
+  CONSTRAINT `FK_publication_collaborator_rel_PublicationID` FOREIGN KEY(`PublicationID`) REFERENCES `publication` (`PublicationID`),
+  CONSTRAINT `FK_publication_collaborator_rel_PublicationCollaboratorID` FOREIGN KEY(`PublicationCollaboratorID`) REFERENCES `publication_collaborator` (`PublicationCollaboratorID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+-- Publication - Variable of Interest  relational table
+DROP TABLE IF EXISTS publication_parameter_type_rel;
+CREATE TABLE `publication_parameter_type_rel` (
+    `PublicationID` int(10) unsigned NOT NULL,
+    `ParameterTypeID` int(10) unsigned NOT NULL,
+    CONSTRAINT `PK_publication_parameter_type_rel` PRIMARY KEY (PublicationID, ParameterTypeID),
+    CONSTRAINT `FK_publication_parameter_type_rel_PublicationID` FOREIGN KEY (`PublicationID`) REFERENCES `publication` (`PublicationID`),
+    CONSTRAINT `FK_publication_parameter_type_rel_ParameterTypeID` FOREIGN KEY (`ParameterTypeID`) REFERENCES `parameter_type` (`ParameterTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+DROP TABLE IF EXISTS publication_test_names_rel;
+CREATE TABLE `publication_test_names_rel` (
+    `PublicationID` int(10) unsigned NOT NULL,
+    `TestNameID` int(10) unsigned NOT NULL,
+    CONSTRAINT `PK_publication_test_names_rel` PRIMARY KEY(`PublicationID`, `TestNameID`),
+    CONSTRAINT `FK_publication_test_names_rel_PublicationID` FOREIGN KEY (`PublicationID`) REFERENCES `publication` (`PublicationID`),
+    CONSTRAINT `FK_publication_test_names_rel_TestNameID` FOREIGN KEY (`TestNameID`) REFERENCES `test_names` (`ID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+-- Publication Uploads
+DROP TABLE IF EXISTS publication_upload_type;
+CREATE TABLE `publication_upload_type` (
+  `PublicationUploadTypeID` int(2) unsigned NOT NULL AUTO_INCREMENT,
+  `Label` varchar(255) NOT NULL,
+  CONSTRAINT `PK_publication_upload_type` PRIMARY KEY (`PublicationUploadTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+INSERT INTO publication_upload_type (`Label`) VALUES ('Paper');
+INSERT INTO publication_upload_type (`Label`) VALUES ('Poster');
+INSERT INTO publication_upload_type (`Label`) VALUES ('Presentation');
+INSERT INTO publication_upload_type (`Label`) VALUES ('Other');
+
+DROP TABLE IF EXISTS publication_upload;
+CREATE TABLE `publication_upload` (
+    `PublicationUploadID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+    `PublicationID` int(10) unsigned NOT NULL,
+    `PublicationUploadTypeID` int(2) unsigned NOT NULL,
+    `URL` varchar(255) NOT NULL,
+    `Version` varchar(255),
+    `Citation` text,
+    CONSTRAINT `PK_publication_upload` PRIMARY KEY (`PublicationUploadID`),
+    CONSTRAINT `UK_publication_upload_URL` UNIQUE (URL),
```

Comment:
Is this an external URL (Pubmed) or the URL on (e.g.) *.loris.ca where the paper will be hosted/downloadable (prob relative to a Config setting?) ? 
Adding an External URL field is not a bad idea for preprints, the official publisher's site, etc. 

* per my comment in the Readme, this is a very different sense of "publication" from "project proposal"